src/Raven.Server/Documents/Patch/V8/DictionaryDispose.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using V8.Net;
+
+namespace Raven.Server.Documents.Patch.V8
+{
+    public class DictionaryDisposeKeyIHV8<TValue> : DictionaryDisposeKeyHandle<InternalHandle, TValue>
+    {
+    }
+
+    public class DictionaryDisposeValueIHV8<TKey> : DictionaryDisposeValueHandle<TKey, InternalHandle>
+    {
+    }
```

Comment:
With V8.Net we deal with JS values by means of InternalHandle struct to avoid GC participation for better performance. When we store InternalHandle we should 'clone' it (increment reference count) and after use we should dispose it (decrement reference count and on getting zero dispose of the native JS value).
The generic JsHandle is struct as well to keep this approach.

With DictionaryDispose* helpers we get the centralization of this functionality when storing those handles as keys or values. RavenEtlScriptRun has seven of such dictionaries. Without them it turns to a mess and a high risk of multiple JS value leakages, that are really hard to find.